vanilla-tests/pom.xml

```diff
@@ -14,11 +14,28 @@
   <artifactId>autorest-vanilla-tests</artifactId>
   <version>1.0.0-SNAPSHOT</version>
 
+  <repositories>
+    <repository>
+      <id>azuresdkmaven</id>
+      <url>https://azsdkartifacts.blob.core.windows.net/maven</url>
+    </repository>
+    <repository>
+      <id>azure-sdk-for-java</id>
+      <url>https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-java/maven/v1</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
   <dependencies>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.5.0</version>
+      <version>1.10.0-alpha.20201022.0</version>
```

Comment:
Upgrading 1.5.0 to 1.9.0 actually breaks a few tests (likely due to changed azure-core behavior on `null` or something).

Reply:
Yes, there are some changes to azure-core to use the UTC timezone. I will update the tests in a follow-up PR to fix this. For now, these tests are not related to the changes in the PR and have reverted the test dependency to use 1.5.0.